frontend/app/ui_components/expandable-search.js

```diff
@@ -46,7 +46,7 @@ module.exports = function(ENTER_KEY) {
         if (scope.collapsed === true) {
           setCollapsed(false);
           // Avoid access key focus
-          setTimeout(function() { input.focus(); });
+          setTimeout(function() { input.focus(); }, 20);
```

Comment:
Regardless whether this solution is working or not, since this is such a brittle and awkward looking fix I would add a comment about "why the hell do we wait 20 ms here?"

Especially since it is something we will never cover using unit tests...